phpBB/includes/functions.php

```diff
@@ -1492,10 +1492,9 @@ function tracking_unserialize($string, $max_depth = 3)
 * @return string The corrected url.
 *
 * Examples:
-* <code>
-* append_sid("{$phpbb_root_path}viewtopic.$phpEx?t=1&amp;f=2");
-* append_sid("{$phpbb_root_path}viewtopic.$phpEx", 't=1&amp;f=2');
-* append_sid("{$phpbb_root_path}viewtopic.$phpEx", 't=1&f=2', false);
+* <code> append_sid("{$phpbb_root_path}viewtopic.$phpEx?t=1");
```

Comment:
Not necessary, but it keeps the examples aligned with the expected usage...

Reply:
OK, I have changed the last example

Reply:
There is no difference between `&amp;` and `?` since the first codes the the second, right?
So a way keep reasonable examples with this would be:
```
append_sid("{$phpbb_root_path}viewtopic.$phpEx?t=1&start=10");
append_sid("{$phpbb_root_path}viewtopic.$phpEx?t=1&amp;start=10");
```
I.e. show thread `1` starting with the 10th post at the top.

The last example here conflicts with the idea of this PR:
```
append_sid("{$phpbb_root_path}viewtopic.$phpEx", array('t' => 1, 'f' => 2));
```
Again it defines thread and forum IDs, which is redundant. As well here the `start` parameter could serve for the example:
```
append_sid("{$phpbb_root_path}viewtopic.$phpEx", array('t' => 1, 'start' => 10));
```